spec/tutum_spec.rb

```diff
@@ -1,15 +1,16 @@
 require_relative './spec_helper'
 
-test_username = ENV['TUTUM_USERNAME']
-test_api_key = ENV['TUTUM_API_KEY']
+test_username = ENV['TUTUM_USERNAME'] || "tutum_username"
+test_api_key = ENV['TUTUM_API_KEY'] || "tutum_api_key"
 test_tutum_auth = ENV['TUTUM_AUTH'] || "ApiKey #{test_username}:#{test_api_key}" #best way to handle this?
+json_opts = {}
 
 describe Tutum do
 
   describe "Existing initialize api" do
 
     subject do
-      Tutum.new(test_username, test_api_key)
+      Tutum.new(test_username, test_api_key, json_opts)
```

Comment:
I think this test should not have json_opts, and should test to ensure that json_opts was set to `{}`